src/aioamazondevices/api.py

```diff
@@ -940,7 +944,7 @@ async def _send_message(
         self,
         device: AmazonDevice,
         message_type: str,
-        message_body: str,
+        message_body: str | float,
```

Comment:
```suggestion
        message_body: str,
```

I would avoid changing the method typing